drivers/scheduler/k8s/k8s.go

```diff
@@ -2085,10 +2085,8 @@ func (k *K8s) WaitForRunning(ctx *scheduler.Context, timeout, retryInterval time
 		var terminatingPods []string
 		pods, err := k.getPodsForApp(ctx)
 		// sadly, getPodsForApp returns an error if there are no pods
-		if err != schederrors.ErrPodsNotFound {
+		if err != nil {
 			return nil, false, nil
```

Comment:
This returns success after there is an error while getting the pods. What if there was an error such as API server was busy, or communication error etc.? We would falsely think that all the pods are gone?

Is it possible to ignore only the specific error that you are hitting in the backup scenario?